src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -94,6 +136,7 @@
                 contentTypesMap,
                 totalCount: subItemsList.ChildrenCount,
                 udwConfigBulkMoveItems,
+                onShowBulkOperationFailedModal: showBulkOperationFailedModal,
```

Comment:
Please, read it out loud: `onShowBulkOperationFailedModal: showBulkOperationFailedModal`. Does it make sense to you? Maybe you can simplify it and avoid a feeling of duplication?

Reply:
So, `onShowBulkOperationFailedModal: showBulkOperationFailedModal` -> `showBulkOperationFailedModal, ` ?